src/inputSystem.ml

```diff
@@ -28,7 +28,7 @@ module SVar = StateVar
 module SVM = SVar.StateVarMap
 
 type _ t =
-| Lustre : (LustreNode.t S.t * LustreGlobals.t) -> LustreNode.t t
+| Lustre : (LustreNode.t S.t * LustreGlobals.t * LustreAst.declaration list) -> LustreNode.t t
```

Comment:
Please write a comment documenting how each element of the tuple is used for, in particular, the third one.